wiki/People/Beatmap_Nominators/en.md

```diff
@@ -59,12 +59,13 @@ Also note: the game modes columns listed below indicates the game mode(s) in whi
 | [-Sh1n1-](https://osu.ppy.sh/u/1957638)           | ![No][false] | ![Yes][true] | ![Yes][true] | ![No][false] | Spanish                  |
 | [Absolute Zero](https://osu.ppy.sh/u/4847256)     | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] |                          |
 | [alienflybot](https://osu.ppy.sh/u/636114)        | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Cantonese, Chinese       |
+| [Aloda](https://osu.ppy.sh/u/1190127)             | ![Yes][true] | ![Yes][true] | ![No][false] | ![No][false] |                         |
 | [Arrival](https://osu.ppy.sh/u/1694000)           | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | French                   |
 | [Asherz007](https://osu.ppy.sh/u/9014047)         | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
 | [Battle](https://osu.ppy.sh/u/4037545)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
 | [Benny-](https://osu.ppy.sh/u/4023183)            | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Norwegian                |
 | [Bonsai](https://osu.ppy.sh/u/987334)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                   |
-| [celerih](https://osu.ppy.sh/u/4696296)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                         |
+| [celerih](https://osu.ppy.sh/u/4696296)           | ![Yes][true] | ![No][false] | ![Yes][false] | ![No][false] | French                 |
```

Comment:
for the one you changed, change the `[false]` to `[true]`

Reply:
oops fixed